learn.md

```diff
@@ -10,7 +10,7 @@ permalink: /learn/
 <!-- <a href="" class="cGreenLinkArrow">Take Quick Tour</a> -->
 </div>
 <div class="col-sm-12 col-md-12 cBallerinaLearnTitleWrapper">
-   <h2>Getting set up to dance</h2>
+   <h2>Get started</h2>
```

Comment:
```suggestion
   <h2>Getting Started</h2>
```

Reply:
We need to say "Get Started" to be consistent with the form of the other title "Learn the language and the libraries". We do not have the -ing form in that.

Reply:
Ack.

Reply:
With this change, it will look like we have a topic named "Getting Started" in learn page [1] and under that we have 3 items, where one is linking to another "Getting Started" page (Which is confusing and duplicated topics IMO)

[1] https://ballerina.io/learn/

Reply:
Did the below changes to address this:

<img width="1197" alt="Screenshot 2019-11-26 at 10 39 32" src="https://user-images.githubusercontent.com/11707273/69601479-49c4c000-103a-11ea-8607-f0383798a0a7.png">
<img width="1230" alt="Screenshot 2019-11-26 at 10 39 45" src="https://user-images.githubusercontent.com/11707273/69601483-4d584700-103a-11ea-8e20-e982d4ac2f6a.png">